python/cudf_polars/tests/test_groupby.py

```diff
@@ -72,7 +74,28 @@ def test_groupby(df: pl.LazyFrame, maintain_order, keys, exprs):
     if not maintain_order:
         sort_keys = list(q.schema.keys())[: len(keys)]
         q = q.sort(*sort_keys)
-    # from cudf_polars.dsl.translate import translate_ir
-    # ir = translate_ir(q._ldf.visit())
-    # from IPython import embed; embed()
+
     assert_gpu_result_equal(q, check_exact=False)
+
+
+def test_groupby_len(df, keys):
+    q = df.group_by(*keys).agg(pl.len())
+
+    # TODO: polars returns UInt32, libcudf returns Int32
```

Comment:
I don't think this will ever change in libcudf, so what's the TODO here?

Reply:
Decide if we want to support exact dtype-matching, and then what the resolution to https://github.com/rapidsai/cudf/issues/15852 is, and if it is "libcudf is not going to do anything", insert unary casts for mismatches.